processor/src/fast/basic_block.rs

```diff
@@ -44,31 +45,70 @@ impl FastProcessor {
         self.execute_before_enter_decorators(node_id, current_forest, host)?;
 
         // Corresponds to the row inserted for the BASIC BLOCK operation added to the trace.
-        self.increment_clk(tracer);
+        self.increment_clk(tracer, stopper).map_break(|_| {
+            BreakReason::Stopped(Some(Continuation::ResumeBasicBlock {
+                node_id,
+                batch_index: 0,
+                op_idx_in_batch: 0,
+            }))
+        })?;
 
-        let mut batch_offset_in_block = 0;
-        let mut op_batches = basic_block_node.op_batches().iter();
-
-        // execute first op batch
-        if let Some(first_op_batch) = op_batches.next() {
+        // execute first batch
+        if let Some(first_op_batch) = basic_block_node.op_batches().first() {
             self.execute_op_batch(
                 basic_block_node,
                 first_op_batch,
                 0,
-                batch_offset_in_block,
-                program,
+                0,
+                0,
                 host,
                 continuation_stack,
                 current_forest,
                 tracer,
+                stopper,
             )
             .await?;
-            batch_offset_in_block += first_op_batch.ops().len();
         }
 
+        // execute the rest of the batches
+        self.execute_basic_block_node_from_batch(
+            basic_block_node,
+            node_id,
+            1,
+            host,
+            continuation_stack,
+            current_forest,
+            tracer,
+            stopper,
+        )
+        .await
+    }
+
+    /// Executes the give basic block node starting from the RESPAN preceding the specified batch.
+    #[expect(clippy::too_many_arguments)]
```

Comment:
This should be configured in the `.clippy.toml` file at the root of the repo.

Reply:
Done